### ISSUE TITLE
Medaka : Installation never finish, medaka conda dependency remain unresolved

### ISSUE THREAD

**Issue by BioinformaticsPlatformWIV-ISP** [galaxyproject/tools-iuc#4915]
Hello,

We have made several attempts to install the medaka_suite into our local galaxy instance and none where successful. The issue occurs with every single tools of the suite : medaka_snp, medaka_concensus and  medaka_variant.  With all version of the tool, under Galaxy 21.09 and 22.01, python 3.8. 
An entry is created in the tool panel successfully but the installation is always stuck for ages at the dependency step, where it fail to install the medaka dependency trough Conda. Thus the tools is available in the instance but never run : "command not found". 
In addition,  manuals installations freeze the whole galaxy instance :  make it unreachable : "Connection Timout" and force us to restart the server (or kill manually all galaxy process that have a lock).

**Comment by mvdbeek** [comment]
If you have any possibility to run tool dependencies in singularity or docker I would highly recommend that you do that. Conda resolution frequently breaks over time. You can install Conda environments outside of Galaxy (the command Galaxy uses is printed in the logs), it's possible that using mamba would help you there. I don't think there's much we can do on our end.

**Comment by wm75** [comment]
@BioinformaticsPlatformWIV-ISP yep, conda is not able to resolve all dependencies of medaka anymore. I bet you'll be positively surprised by mamba though if you try the manual install with it instead.

**Comment by nsoranzo** [comment]
And the next Galaxy release will use the (experimental) libmamba solver in Conda: https://github.com/galaxyproject/galaxy/pull/14882
